src/actions/sources.js

```diff
@@ -131,13 +131,7 @@ function loadSourceMap(generatedSource) {
         }: Source)
     );
 
-    dispatch({ type: "ADD_SOURCES", sources: originalSources });
-
-    await dispatch(loadSourceText(generatedSource));
-    originalSources.forEach(async source => {
-      await checkSelectedSource(getState(), dispatch, source);
-      checkPendingBreakpoints(getState(), dispatch, source.id);
-    });
+    dispatch(newSources(originalSources));
```

Comment:
this works except it's missing the `loadSourceText(generatedSource)` for the generated source. 

this would work:

```js
 await dispatch(loadSourceText(generatedSource));
 dispatch(newSources(originalSources));
```

the use case is that pending breakpoints need the generated source text to do the syncing...

Reply:
why do we need to load the source text here? we are loading it in newSource after we check the sourcemap, so this is a second check, isnt it?

Reply:
This is a small cleanup. I have another PR that removes the need for `loadSourceText`

We'll fix the timing issues in other PRs.

Reply:
this comment isn't clear to me